rocAL/rocAL_pybind/examples/rocAL_api_pytorch_classification_reader.py

```diff
@@ -39,7 +39,7 @@ def draw_patches(img, idx, layout="nchw", dtype="fp32", device="cpu"):
     if dtype == "fp16":
         image = image.astype('uint8')
     image = cv2.cvtColor(image, cv2.COLOR_RGB2BGR)
-    cv2.imwrite("OUTPUT_IMAGES_PYTHON/FILE_READER/" + str(idx)+"_"+"train"+".png", image * 255)
+    cv2.imwrite("OUTPUT_IMAGES_PYTHON/FILE_READER/" + str(idx)+"_"+"train"+".png", image )
```

Comment:
Check if need to use *255 for normalized images

Reply:
Modified the example to use original mean and std so the multiplying images by 255 is reintroduced